src/App.js

```diff
@@ -1,109 +1,65 @@
-        title, country, countrycode, city or description. Searchterms are case-sensitive and can
-        omit characters. When tags are selected, results are shown in order of relevance. Tags and
-        search can be used in combination.
-      </StyledExplanation>
-      {tags && <TagCluster tags={tags} onTagClick={onTagClick} />}
-      {projects &&
-        projects
-          .filter((project) => project.accordingToSearchTerms ?? project)
-          .slice(0, 20)
-          .map((project, index) => (
-            <Project
-              key={project._id}
-              project={project}
-              onClick={() => toggleDetailOverlay(project, index)}
-            />
-          ))}
-      {projects &&
-        projects.map(
-          (project, index) =>
-            project.expanded && (
-              <Overlay
-                key={project._id}
-                project={project}
-                onBack={() => toggleDetailOverlay(project, index)}
-              />
-            )
-        )}
-    </>
+    <div {...swipe}>
+      <Router>
+        {isNavigationVisible && <Title>UNFINISHED</Title>}
+        {isNavigationVisible && <Navigation onClick={() => setNavigationsVisible(false)} />}
+        <ContentPlacementStyled
+          visible={isNavigationVisible ? 1 : 0}
+          style={slide}
+          onClick={() => isNavigationVisible && setNavigationsVisible(false)}
+        >
+          <Switch>
+            <Route path='/opportunities'>
+              <Opportunities onBack={() => setNavigationsVisible(true)} />
+            </Route>
+            <Route path='/create'>
+              <CreateProject onBack={() => setNavigationsVisible(true)} />
+            </Route>
+            <Route path='/my-projects'>
+              <UserProjects onBack={() => setNavigationsVisible(true)} />
+            </Route>
+            <Route path='/'>
+              <Dashboard onBack={() => setNavigationsVisible(true)} />
+            </Route>
+          </Switch>
+        </ContentPlacementStyled>
+      </Router>
+    </div>
   )
-  // Mehr Anzeigen button einbauen
-  function toggleDetailOverlay(project, index) {
-    setProjects([
-      ...projects.slice(0, index),
-      { ...project, expanded: !project.expanded },
-      ...projects.slice(index + 1),
-    ])
-    !document.body.classList.contains('overlay') && setScrollPosition(window.scrollY)
-    document.body.classList.toggle('overlay')
-    window.scroll({ top: scrollPosition, left: 0, behavior: 'smooth' })
-  }
-  function onTagClick(tag, index) {
-    setTags([
-      ...tags.slice(0, index),
-      { text: tag.text, applies: !tag.applies },
-      ...tags.slice(index + 1),
-    ])
-  }
-  function onSearch(event) {
-    setSearchTerm(event.target.value || ' ')
-  }
 }
-
-const StyledExplanation = styled.p`
-  margin: 20px;
-  color: darkgrey;
-  font-size: 80%;
-  text-align: center;
+Opportunities.propTypes = {
```

Comment:
Why do you have `Opportunities.propTypes` in your App.js and not in Opportunities.js?

Reply:
no idea, fixed it